convert2rhel/pkghandler.py

```diff
@@ -505,6 +592,21 @@ def print_pkg_info(pkgs):
     return pkg_table
 
 
+def _get_package_repository(pkg):
+    """Get package repository information."""
+    package = get_pkg_nvra(pkg.nevra) if isinstance(pkg, PackageInformation) else get_pkg_nvra(pkg)
+    output, retcode = utils.run_subprocess(
+        ["repoquery", "--quiet", "-q", package, "--qf", "C2R %{REPOID}"],
+        print_cmd=False,
+        print_output=False,
+    )
+
+    if "C2R" not in output or retcode != 0:
+        return "N/A"
+
+    return output.lstrip("C2R ")
+
+
```

Comment:
`get_pkg_nvra` and `get_pkg_nevra` can be changed to extract the nevra from PackageInfo objects as well as the yum/dnf objects that represent packages.  Maybe with a helper function like this?
``` python
def _get_nevra_from_pkg_obj(pkg_obj):
    if isinstance(pkg_obj, PackageInformation):
        return pkg_obj.nevra
    return PackageNevra(name=pkg_obj.name, 
                        epoch=pkg_obj.epoch,
                        version=pkg_obj.version,
                        release=pkg_obj.release,
                        arch=pkg_obj.arch)

def get_pkg_nvra(pkg_obj):
    nevra = _get_nevra_from_pkg_obj(pkg_obj)
    # [...]
```

Reply:
added the help function